priv/script.sh

```diff
@@ -2,441 +2,489 @@
-        ;;
-    "PostgreSQL")
-        if [ "$os_type" = "macOS" ]; then
-            brew install gcc readline zlib curl ossp-uuid
-        else
-            case "$package_manager" in
-            "apt")
-                sudo apt-get update
-                sudo apt-get install -y linux-headers-generic build-essential libssl-dev libreadline-dev zlib1g-dev libcurl4-openssl-dev uuid-dev icu-devtools
-                ;;
-            "dnf")
-                sudo dnf groupinstall -y "Development Tools"
-                sudo dnf install -y kernel-headers openssl-devel readline-devel zlib-devel libcurl-devel libuuid-devel libicu-devel
-                ;;
-            "pacman")
-                sudo pacman -Sy --noconfirm linux-headers base-devel openssl readline zlib curl util-linux icu
-                ;;
-            "apk")
-                sudo apk add --no-cache linux-headers build-base openssl-dev readline-dev zlib-dev curl-dev util-linux-dev icu-dev
-                ;;
-            esac
-        fi
-        mise use -g -y postgres@$postgres_version
-        ;;
-    "Xcode Command Line Tools")
-        xcode-select --install
-        ;;
+    Homebrew | Erlang)
+      printf "Installing %s... (this might take a while)\n" "$1"
+      ;;
     *)
-        printf "Invalid name argument on install: %s\n" "$1"
-        exit 1
-        ;;
+      printf "Installing %s...\n" "$1"
+      ;;
     esac
+
+    install "$1"
+  fi
 }
 
-maybe_install() {
-    if already_installed "$1"; then
-        printf "%s is already installed. Skipping...\n" "$1"
-    else
-        printf "Installing %s...\n" "$1"
+install_shell_scripts() {
+  mkdir -p "$phx_tools_path"
+
+  local_bin=\"\$HOME\"/.local/bin
+
+  cat >"$phx_tools_sh_path" <<SH
+#!/bin/sh
+
+# This file is managed by phx.tools and should not be edited by hand.
+SH
+
+  if [ -n "$homebrew_path" ]; then
+    cat >>"$phx_tools_sh_path" <<SH
+
+eval "\$("$homebrew_path"/brew shellenv)"
+SH
+  fi
+
+  cat >>"$phx_tools_sh_path" <<SH
+
+if ! command -v mise >/dev/null 2>/dev/null && [ -x $local_bin/mise ]; then
+  export PATH="$local_bin:\$PATH"
+fi
+
+eval "\$(mise activate "\$(basename \$SHELL)")"
+SH
+
+  cat >"$phx_tools_fish_path" <<FISH
+# This file is managed by phx.tools and should not be edited by hand.
+FISH
+
+  if [ -n "$homebrew_path" ]; then
+    cat >>"$phx_tools_fish_path" <<FISH
+
+$homebrew_path/brew shellenv | source
+FISH
+  fi
+
+  cat >>"$phx_tools_fish_path" <<FISH
+
+if ! command -sq mise && test -x $local_bin/mise
+  fish_add_path --global --path --move $local_bin
+end
 
-        if [ "$1" = "Homebrew" ] || [ "$1" = "Erlang" ]; then
-            printf "This might take a while.\n"
-        fi
+if ! functions -q __mise_env_eval && status is-interactive
+  mise activate fish | source
+end
+FISH
 
-        printf "\n"
-        install "$1"
+  sh_source_line="source '$phx_tools_sh_path'"
+
+  if has bash && [ -f "$HOME"/.bashrc ]; then
```

Comment:
Here we update `$HOME/.bashrc` to source `~/.config/phx_tools/phx_tools.sh`, but only if bash is installed and `~/.bashrc` exists.